frontend/src/containers/about/logos-grid/types.ts

```diff
@@ -1,7 +1,8 @@
 export type LogoType = {
   logo: string;
   alt: string;
-  description: string;
+  link?: string;
+  description?: string;
   dimensions: [number, number];
```

Comment:
Instead of coming up with your own types, I do recommend always referring to the types of the element/component you are using. For example: we both agree `alt` is a string and the type of `alt` in `Image`, why not just using its typing instead of coming up with your own (and the later maintenance).

Most of components and elements we use already have their own typing, we just need to follow it. The only problem I can find with your approach if, if the future `alt` changes to `string[]`, you will not notice ad your type is fine (it still says `string`), but the actually the image is waiting for `string[]`. If you would have pointed to its type, you would have noticed in the code as it would fail :)

(Ping me if the explanation is not clear or whatever)

![image](https://github.com/Vizzuality/skytruth-30x30/assets/999124/79ceb80f-6aa5-49a8-ad32-866c80628a18)

This is just an example of extracting the type. There are other valid ways.